openmc/mgxs/mgxs.py

```diff
@@ -2190,7 +2196,7 @@ class TransportXS(MGXS):
     tally_keys : list of str
         The keys into the tallies dictionary for each tally used to compute
         the multi-group cross section
-    estimator : {'tracklength', 'analog'}
+    estimator : {'tracklength', 'collision', 'analog'}
```

Comment:
I'm confused about this one - this cannot be tracklength right? If so, shouldn't the docstring reflect that the tallies can only be analog or collision?